applications/ipc_radio/README.rst

```diff
@@ -60,35 +60,53 @@ Configuration
 
 |config|
 
-Application
-===========
+Configuration options
+=====================
 
 You can set the supported radio configurations using the following Kconfig options:
 
-* :kconfig:option:`CONFIG_IPC_RADIO_BT` - For the Bluetooth Low Energy serialization.
-* :kconfig:option:`CONFIG_IPC_RADIO_802154` - For the IEEE 802.15.4 serialization.
+.. _CONFIG_IPC_RADIO_BT:
+
+CONFIG_IPC_RADIO_BT
+   This option enables the IPC Bluetooth Low Energy radio serialization.
+
+.. _CONFIG_IPC_RADIO_802154:
+
+CONFIG_IPC_RADIO_802154
+   This option enables the IPC IEEE 802.15.4 radio serialization.
+
+You can select the Bluetooth Low Energy serialization using the following Kconfig options:
+
+.. _CONFIG_IPC_RADIO_BT_SER:
+
+CONFIG_IPC_RADIO_BT_SER
+   This option selects the type of the IPC Bluetooth radio serialization.
+
+.. _CONFIG_IPC_RADIO_BT_HCI_IPC:
 
+CONFIG_IPC_RADIO_BT_HCI_IPC
+   This option enables the Bluetooth HCI serialization.
 
-You can select the Bluetooth Low Energy serialization using the ``CONFIG_IPC_RADIO_BT_SER`` Kconfig choice:
+.. _CONFIG_IPC_RADIO_BT_RPC:
 
-* :kconfig:option:`CONFIG_IPC_RADIO_BT_HCI_IPC` - For the Bluetooth HCI serialization.
-* :kconfig:option:`CONFIG_IPC_RADIO_BT_RPC` - For the Bluetooth host API serialization.
+CONFIG_IPC_RADIO_BT_RPC
+   This option enables the Bluetooth host API serialization over RPC.
 
 The Bluetooth Low Energy and IEEE 802.15.4 functionalities can operate simultaneously and are only limited by available memory.
 
 Sysbuild Kconfig options
 ========================
 
-To enable the firmware, use the sysbuild configuration ``SB_CONFIG_NETCORE_IPC_RADIO``.
+To enable the firmware, use the sysbuild configuration :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO`.
 
 You can set the supported radio configurations using the following sysbuild Kconfig options:
 
-* ``SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC``
-* ``SB_CONFIG_NETCORE_IPC_RADIO_BT_RPC``
-* ``SB_CONFIG_NETCORE_IPC_RADIO_IEEE802154``
+* :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC`
+* :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO_BT_RPC`
+* :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO_IEEE802154`
 
 .. note::
-   For |NCS| samples and applications, use the ``SB_CONFIG_NRF_DEFAULT_IPC_RADIO`` sysbuild configuration to enable the firmware instead of ``SB_CONFIG_NETCORE_IPC_RADIO`` (which should only be used for production).
+   For |NCS| samples and applications, use the :kconfig:option:`SB_CONFIG_NRF_DEFAULT_IPC_RADIO` sysbuild configuration to enable the firmware instead of :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO` (which must only be used for production).
```

Comment:
@dchat-nordic, where is this SB_CONFIG_NRF_DEFAULT_IPC_RADIO option? Not found in the Kconfig reference and thus, not creating a link. Or is it just wrong?

Reply:
The SB_* Kconfig are actually Sysbuild Kconfigs - most of them are in sdk-nrf/sysbuild/Kconfig.* - The ones from the line are in Kconfig.netcore. 
I don't know if those Kconfigs are actually indexed by the documentation build, that's backticks were used before.